### PR TITLE
feat: fine-grain resource config at skeleton level

### PR DESCRIPTION
**Pull request by justinthelaw**
- Add README details
- Moved GPU, CPU, and Memoray resource control to skeleton
- GPU enablement and resources always prompted

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -25,21 +25,16 @@ serviceAccount:
 
 podAnnotations: {}
 
-podSecurityContext: {}
-  # fsGroup: 2000
+podSecurityContext:
+  runAsNonRoot: true
 
 securityContext: {}
```
Comment (YrrepNoj) [inline chart/values.yaml:31]:
```suggestion
securityContext:
  runAsUser: 65532
  runAsGroup: 65532
  fsGroup: 65532
```

File: zarf.yaml [hunk 2]
```diff
@@ -15,18 +15,31 @@ constants:
...
-  - name: LIMITS_CPU
-    default: "0"
-  - name: LIMITS_MEMORY
-    default: "0"
-  - name: LIMITS_GPU
-    default: "0"
+  - name: GPU_ENABLED
+    description: Optionally turn on GPU inferencing (true/false)
+    default: "false"
+    prompt: true
```
Comment (YrrepNoj) [inline zarf.yaml:21]: Here and below; Can we remove the `prompt: true`?

As we start adopting more UDS patterns we are going to move away from Zarf variables and want to start moving away from prompting/expecting them.

**Review by YrrepNoj — CHANGES_REQUESTED** [review]

File: zarf.yaml (hunk 2 above)
Comment (justinthelaw) [reply zarf.yaml:21]: Yessir! Also removed form the REQUESTS and LIMITS

File: zarf.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 kind: ZarfPackageConfig
 metadata:
   name: leapfrogai-model
-  version: 0.4.1
+  version: 0.5.0
```
[the remark this answers is not in the document]
Comment (justinthelaw) [reply zarf.yaml]: I like that! Done

**Review by YrrepNoj — APPROVED** [review]

**Pull request merged**